flytekit/deck/renderer.py

```diff
@@ -48,3 +49,10 @@ class ArrowRenderer:
     def to_html(self, df: "pyarrow.Table") -> str:
         assert isinstance(df, pyarrow.Table)
         return df.to_string()
+
+
+class MarkdownRenderer:
```

Comment:
we already have this in the [plugin](https://github.com/flyteorg/flytekit/blob/b4de2f845972967746babbeb8be645025d214086/plugins/flytekit-deck-standard/flytekitplugins/deck/renderer.py#L64-L73). if we want to move to core, the onc in the plugin can be removed.

Reply:
I deprecated the previous `MarkdownRenderer` here: [`4e93235` (#2246)](https://github.com/flyteorg/flytekit/pull/2246/commits/4e93235ab3f3177ca93fe52e38964cb1b3a034a4)

Given that `FlyteDeck` is beta, I am okay with removing it from the plugin as well.

What do you prefer?

Reply:
makes sense, let's deprecate first